app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="home_cont">
+  <div class="splashlogo_cont">
+  <%= image_tag 'splashlogo3.gif', alt: 'splash logo', class: 'custom-class', id: 'splashlogo' %>
+
+  </div>
+  <div class="regbtns_cont">
+    <%= link_to 'Sign up', new_user_registration_path, class: 'regbtns' %>
+    <%= link_to 'Login', new_user_session_path, class: 'regbtns' %>
+  </div>
```

Comment:
- [ ] While you did super with the splash page design, kindly note that you are required to maintain the visual cues that were provided in the [Behance design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).

**Below is how your design looks like at the moment:**

![image](https://github.com/geekelo/cash-in-out/assets/86318284/2291a23f-9a46-458d-8231-b2098f7ba5a2)

**Kindly note that this is the desired design of the buttons having them scale the whole width of the page:**

![image](https://github.com/geekelo/cash-in-out/assets/86318284/f4896674-cdcb-45a4-aa69-baf9a6e3708e)